### PR TITLE
Remove closing tag from void elements

### PR DESCRIPTION
**Pull request by NiedziolkaMichal**
I have removed closing tags from all void elements, to make it consistent across all html files. All void elements were checked, but there were problems only with: br, hr, img, track, input, area

### CODE REVIEW

File: live-examples/css-examples/positioned-layout/position.html [hunk 1]
```diff
@@ -41,8 +41,8 @@
             <p class="clear">To see the effect of <code>sticky</code> positioning, select the <code>position: sticky</code> option and scroll this container.</p>
             <p>The element will scroll along with its container, until it is at the top of the container (or reaches the offset specified in <code>top</code>), and will then stop scrolling, so it stays visible.</p>
             <p>The rest of this text is only supplied to make sure the container overflows, so as to enable you to scroll it and see the effect.</p>
-            <hr/>
             <p>Far out in the uncharted backwaters of the unfashionable end of the western spiral arm of the Galaxy lies a small unregarded yellow sun. Orbiting this at a distance of roughly ninety-two million miles is an utterly insignificant little blue green planet whose ape-descended life forms are so amazingly primitive that they still think digital watches are a pretty neat idea.</p> 
+            <hr>
```
Comment (wbamberg) [inline live-examples/css-examples/positioned-layout/position.html]: Why did you move this?

**Review by wbamberg — COMMENTED** [review]
Nice!

Just one question.

Also, I like this and will merge it, but also see: https://github.com/mdn/mdn-community/discussions/242 - if we ever run Prettier on this repo we'll have a problem here.

File: live-examples/css-examples/positioned-layout/position.html (hunk 1 above)
Comment (NiedziolkaMichal) [reply live-examples/css-examples/positioned-layout/position.html]: It looks to be a mistake, thanks for noticing. I have made a commit with a fix.

**Review by wbamberg — APPROVED** [review]
👍 thank you @NiedziolkaMichal !
